apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.GenerateClient/RoslynHelpers.cs

```diff
@@ -112,9 +112,9 @@ internal static NameSyntax Of(this NameSyntax typeName, params TypeSyntax[] type
                 case SyntaxKind.QualifiedName:
                     return ((QualifiedNameSyntax)typeName).Of(typeArguments);
                 case SyntaxKind.IdentifierName:
-                    return GenericName(((IdentifierNameSyntax)typeName).Identifier).Of(typeArguments);
+                    return ((IdentifierNameSyntax)typeName).Of(typeArguments);
```

Comment:
Just as a thought, could you use pattern matching rather than all the casting?

Reply:
Yup, that's cleaner. Thanks.